OpenEdXMobile/src/main/java/org/edx/mobile/view/adapters/DiscussionCommentsAdapter.java

```diff
@@ -121,24 +130,33 @@ public void onBindViewHolder(RecyclerView.ViewHolder viewHolder, int position) {
             iconDrawable = new IconDrawable(context, FontAwesomeIcons.fa_comment)
                     .sizeRes(context, R.dimen.edx_small)
                     .colorRes(context, R.color.edx_brand_gray_base);
+            holder.discussionCommentCountReportTextView.setVisibility(View.VISIBLE);
```

Comment:
It's weird that we're using the same view holder for both the response header and the comment items, which is causing the same view to be recycled between them, requiring us to write this redundant logic. Defining them as separate view types in the adapter would fix this issue. It's up to you if you wanna do that though.